src/whisk.rs

```diff
@@ -324,14 +328,14 @@ mod tests {
     }
 
     #[test]
-    fn serde_g1_rand() {
-        let p_bytes = hex::decode("6d4761a01a6aa320db42b47ebe47b1fb7f7ab3925c4b1a2c6de3a15e40976596cad444ea4216d467d297ad2081107192").unwrap();
```

Comment:
I'm guessing this was using a test vector generated from arkworks since it worked with arkworks 0.3 -- That version was not compliant with the specs however see https://github.com/arkworks-rs/algebra/issues/257

I changed it to use the generator which should be easier to calibrate across different implementations